docs/SlavaUkraini/examples/video-overlay/index.md

```diff
@@ -1,19 +1,79 @@
 ---
 layout: tutorial_v2
-title: Leaflet on Mobile
+title: Overlay
 ---
 
-## Video on webpages
+# Overlay - Tutorial
```

Comment:
This should be an h2 (`##`), other headings will need to be adjusted accordingly.

Reply:
got it, now I changed all headers for 1 level as it was before.
Before I tried to make it 1 level up since it was hard to distinguish #### (h4) from a simple paragraph